packages/pangea-sdk/pangea/exceptions.py

```diff
@@ -95,3 +95,21 @@ class TreeNotFoundException(AuditAPIException):
 
 class BadOffsetException(AuditAPIException):
     """Bad offset in results search"""
+
+
+# Vault SDK specific exceptions
+class VaultExceptions(PangeaException):
```

Comment:
Not sure in case of Vault, but in Audit I prefer to have AuditException and AuditAPIException to split when an exception it's because of an error in the API request, and when something fails locally, like event signature (if signed locally) or something else.
Agree with renaming in singular as VaultException